packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -88,6 +88,12 @@ class RunCommand extends RunCommandBase {
         defaultsTo: false,
         negatable: false,
         help: 'Start in a paused mode and wait for a debugger to connect.');
+    argParser.addFlag('enable-software-rendering',
+        defaultsTo: false,
+        negatable: false,
+        help: 'Enable rendering using the Skia software backend. This is useful\n'
+              'when testing Flutter on emulators. By default, Flutter will\n'
+              'attempt to either use OpenGL or Vulkan.');
```

Comment:
nit: suggest ending the last sentence with "and will fallback to software rendering when neither are available."